{"name": "Проверка создания заказа с некорректно заданным ингредиентом", "status": "passed", "start": 1717767605673, "stop": 1717767605993, "uuid": "22098898-5b04-49e1-8ca7-b6c4b9676f44", "historyId": "ad2923b87d22f3c464091cc531a838d9", "testCaseId": "ad2923b87d22f3c464091cc531a838d9", "fullName": "tests.py.test_order.TestOrderCreating#test_orders_wrong_ingredient_failed", "labels": [{"name": "parentSuite", "value": "tests.py"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestOrderCreating"}, {"name": "host", "value": "DESKTOP-AQPR8BL"}, {"name": "thread", "value": "16524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.py.test_order"}]}